src/api/worker/facades/CalendarFacade.ts

```diff
@@ -179,6 +180,7 @@ export class CalendarFacade {
 				//In this case the user will not be informed about the number of failed events. We considered this is okay because it is not actionable anyways.
 				throw new ConnectionError("Connection lost while saving events")
 			} else {
+				console.log("Could not save events. Number of failed imports: ", failed)
```

Comment:
I feel we should log a bit more info here, we have a list of all the errors that occurred during `setupMultiple` in the `errors` variable